data/css/modules/card/_default.scss

```diff
@@ -209,6 +209,19 @@
                 padding: 20px;
             }
         }
+
+        // Video/audio icon sizing
```

Comment:
The specs were a bit unclear as to what exact size the video overlays should take. I updated the specs with the specific sizes. 
For the Polaroid card, some of them do need widths, because not every card in that height wants the larger play button. For the Post card, yeah, it's only specified by height.